doc/Makefile

```diff
@@ -42,8 +49,32 @@ info: $(addprefix ../info/,$(SGMLS:.sgml=.info))
 ../info/%.info: %.sgml | ../info
 	@cd ../info && linuxdoc -B info ../doc/$<
 
+.PRECIOUS: ../docwrk/%.tex
+../docwrk/%.tex: %.sgml | ../docwrk
+	@cd ../docwrk && linuxdoc -B latex ../doc/$<
+
+../docwrk/%.dvi: ../docwrk/%.tex
+	@cd ../docwrk && \
+	rm -f $*.make.log; \
+	(logHash=aaa; \
+	lastLogHash=bbb; \
+	count=1; \
+	echo -n "Generating $*.pdf : "; \
+	while [ "$$lastLogHash" != "$$logHash" ]; \
+	  do \
+	   echo -n "..$$count"; \
+	   count=$$(expr $$count + 1);\
+	   $(LATEX) $< >> $*.make.log; \
+	   lastLogHash=$$logHash; \
+	   logHash=$$(md5sum $*.log); \
+	done;)
+	@echo "..done."
```

Comment:
I can imagine that you put quite so effort in this but pretty please don't do such things in cc65 Makefiles. I went to great length to get rid of Uz' shell logic. 

Apart from that https://www.gnu.org/software/make/manual/make.html#Utilities-in-Makefiles doesn't name `while`, `[`, `]`, `do` or `done`.

Reply:
I expect it should work on most systems (on plain "CMD.EXE" Windows the docs aren't being built anyway). "expr" and "md5sum" are part of GNU coreutils which, I guess, are available on any Linux system. I expect "expr" to be unconditionally available on every *nix system since 20 years, at least. The only problem I'm seeing is "md5sum" which is GNU specific. I could workaround by keeping the old file and really comparing it with the new file (using "cmp", available probably since the epoch). So do you want me to make this dance?

Alternatively, I could unconditionally execute latex 5 times or more (5 times is the maximum loop count in the current version with the ca65 docs).